### PR TITLE
Added MoreInfo component and tests

### PR DESCRIPTION
**Pull request by krischarbonneau**
# Description

[LJ-92 "More Information" Links](https://lj-decd.atlassian.net/browse/LJ-92?atlOrigin=eyJpIjoiOTdhNmQ1NzRjZjVmNDE3NmIyZjUyZmE3YmI1OGE0OTIiLCJwIjoiaiJ9 )

Added More Info component that triggers a google search query using the supplied text when clicked.

## Test Instructions

1. Pull in branch
2. Type ```npm run dev```
3. Navigate to [http://localhost:3000/having-a-child](http://localhost:3000/having-a-child) to see all 4 links (as per Stephanie's Figma design) working.

## Meets Definition of Done

- [x] Meets design spec
- [x] unit tests are included

### CODE REVIEW

File: components/atoms/MoreInfo.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const googleSearch = "https://google.ca/search?q="
+    .concat(props.text)
+    .replace(/\s/g, "+");
+
+  return (
+    <div className="pb-2 pt-2">
+      <div className="pb-2 pt-2 px-3 border border-custom-blue-dark rounded text-custom-blue-dark cursor-pointer hover:bg-gray-light-report focus:ring-1 focus:ring-black focus:ring-offset-1 active:bg-gray-active">
+        <Link href={googleSearch}>
+          <a className="block w-full" id={props.id}>
+            <h6>
```
Comment (MarcoGoC) [inline components/atoms/MoreInfo.js:20]: not sure about the H6 here, it implies the page will have h2-h5

Comment (krischarbonneau) [reply components/atoms/MoreInfo.js]: I was following what the Figma designs had but this is a fair point. I removed the h6 tag and instead added styling to make it look like an h6.

File: pages/having-a-child.js [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+import MoreInfo from "../components/atoms/MoreInfo";
 
 export default function lifejourney() {
   return (
     <Layout
       bannerTitle="Having a child"
       bannerText="Lorem ipsum dolor sit amet, consectetur"
-    ></Layout>
+    >
+      <div>
```
Comment (P0NDER0SA) [inline pages/having-a-child.js:10]: maybe we could start now including translation placeholders instead of hardcoded text?

**Review by P0NDER0SA — CHANGES_REQUESTED** [review]
just a note to see if we can get translation placeholders going in this stew....

File: pages/having-a-child.js (hunk 2 above)
Comment (P0NDER0SA) [reply pages/having-a-child.js:10]: we'll chat about this ...  need to see how we're gonna do text in the end

Comment (P0NDER0SA) [reply pages/having-a-child.js:10]: ignore my comments!  different approach

**Review by P0NDER0SA — APPROVED** [review]

**Comment by krischarbonneau** [comment]
> just a note to see if we can get translation placeholders going in this stew....

I know you marked this as resolved already but I went ahead and added translation for the banner and the 4 more info components. I didn't do the report a problem component as that will likely be done in another PR when fixing it up as per Alpha Site's changes.

**Review by MarcoGoC — APPROVED** [review]
It looks good to me, I think Paul should have the last say

File: components/atoms/MoreInfo.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ *  More Information Component
+ */
+
+export default function MoreInfo(props) {
+  const googleSearch = "https://google.ca/search?q="
+    .concat(props.text)
+    .replace(/\s/g, "+");
+
+  return (
+    <div className="pb-2 pt-2">
```
Comment (MarcoGoC) [inline components/atoms/MoreInfo.js:16]: you can remove the outer div and add mt-4 mb-4 to the div below it has the same effect, just a suggestion it is good also as is

Comment (krischarbonneau) [reply components/atoms/MoreInfo.js]: Good suggestion, went ahead and did this

File: components/atoms/MoreInfo.test.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { render, act } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { axe, toHaveNoViolations } from "jest-axe";
+import MoreInfo from "./MoreInfo";
+
+expect.extend(toHaveNoViolations);
+
+const infoText = "Find hot info near me";
```
Comment (pcraig3) [inline components/atoms/MoreInfo.test.js:8]: Local hot info wants to meet *you*! ❤️

File: components/atoms/MoreInfo.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+ */
+
+export default function MoreInfo(props) {
+  const googleSearch = "https://google.ca/search?q="
+    .concat(props.text)
+    .replace(/\s/g, "+");
+
+  return (
+    <div className="mt-4 mb-4 pb-2 pt-2 px-3 border border-custom-blue-dark rounded text-custom-blue-dark cursor-pointer hover:bg-gray-light-report focus:ring-1 focus:ring-black focus:ring-offset-1 active:bg-gray-active">
+      <Link href={googleSearch}>
```
Comment (pcraig3) [inline components/atoms/MoreInfo.js:17]: just an FYI -- nothing wrong with using the Next `Link` component but it doesn't help you at all if you are doing external links. If you are doing links between pages on your site, that's when it speeds things up.

**Review by pcraig3 — APPROVED** [review]
I think it looks really good 👍

File: components/atoms/MoreInfo.js (hunk 5 above)
Comment (krischarbonneau) [reply components/atoms/MoreInfo.js:17]: Good to know!

**Pull request merged**